DOCSP-47144-timeseries-limitations (#197)
mongodb/docs-commandline-tools: source/mongoimport/mongoimport-behavior.txt

``mongoimport`` uses a maximum batch size of 100,000 to
perform bulk insert/upsert operations.

Time Series Collections
~~~~~~~~~~~~~~~~~~~~~~~

> ``mongoimport`` can only import data into a :ref:`time series collection <manual-timeseries-landing>` created before running
> ``mongoimport``. ``mongoimport`` does not support creating time series collections

Reviewer: [2nd Person/Imperative]
The content in this first sentence is great! I think we can make it a bit more clear by shifting to direct commands w/[2nd person & imperative mood](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/).

```suggestion
To import time series data, you must create a :ref:`time series collection <manual-timeseries-landing>`
before running ``mongoimport``. ``mongoimport`` does not support creating time series collections
```

> automatically. To import time series data, run :method:`~db.createCollection()` on your MongoDB
> instance first, then pass in the time series collection you created to ``mongoimport``.

Reviewer: [brevity]
I think we can shorten this last sentence as it seems the `mongoimport` invocation is the same whether or not the data is time series.

```suggestion
automatically. For more information about creating a collection, see :method:`~db.createCollection()`.
```
